lib/IRGen/GenMeta.cpp

```diff
@@ -985,6 +985,16 @@ namespace {
 
       {
         auto *requirement = cast<AbstractFunctionDecl>(func.getDecl());
+        // Check if it's a computed property accessor with a distributed thunk
+        if (auto *accessor = dyn_cast<AccessorDecl>(requirement)) {
```

Comment:
I think the fact that we have to do this points to earlier in the pipeline. There should be no difference between accessor thunks and function thunks, both of the have to have district witness table entries, so I think the fix should be somewhere were the witness list is formed.

Reply:
Thanks Pavel, yeah that makes sense -- let me investigate what causes this and then solve elsewhere 🤔 

There's some TBD issues floating around which may be the same root cause hmm